### DISCUSSION TITLE
Suppressed alerts still coming through - Suricata

### DISCUSSION THREAD

**Discussion by chergett** [Security-Onion-Solutions/securityonion#8011] [Unsupported Versions] [answered]
I have some alerts I'm trying to suppress for zip files from specific IP ranges for two sids.  Below is the relevant code snippet from /opt/so/saltstack/local/pillar/global.sls.

    1100012:
    - suppress:
        gen_id: 1
        track: by_either
        ip: [104.129.206.0/23, 136.226.48.0/23, 136.226.52.0/23]
    1100000:
    - suppress:
        gen_id: 1
        track: by_either
        ip: [104.129.206.0/23, 136.226.48.0/23, 136.226.52.0/23]

I have not seen the "by_either" tracking method used in any examples or documentation thus far so I'm not sure that what I'm doing is allowed.  Even with the above code in place, I'm still getting alerts for those sids from those IP ranges.

SO version: 2.3.120
On-prem with internet access
Installed from iso
Single node 384GB RAM/2 Physical CPUs (16 cores each)
Traffic is sflow coming from core switch.  Average of 20mbps.
so-status shows all services are running
highstate shows no errors
No failures showing on Grid page

**Comment by mnasec** [comment, marked as answer]
As shown in the example/documentation https://docs.securityonion.net/en/2.3/managing-alerts.html?highlight=suppress i suggest to not use brackets and no spaces between the different ip addresses or subnets. Just separate them with a column.
Another hint: After the sid line there are two spaces before the dash. Then it should work from my point of view.

**Reply by chergett** [reply]
Thank you.  I checked the suricata log and didn't notice the errors at first.  Not sure how I missed it but this seems to have worked as the errors are now gone.
